.evergreen/config.yml

```diff
@@ -55,7 +55,7 @@ functions:
         working_dir: "src/${DIR}/${REPO_NAME}"
         binary: bash
         env:
-          atlas: ${workdir}/src/atlas/bin/atlas
+          atlas: ${workdir}/src/atlas/bin/atlas 
```

Comment:
we need a linter. :)

Reply:
Agreed. I'll make a task for it. [PYTHON-4687](https://jira.mongodb.org/browse/PYTHON-4687)